### DISCUSSION TITLE
Next.js로 기술 블로그 만들기: 블로그는 사드세요... 제발 1편 블로그 구색 갖추기(블로그 포스팅 구현하기) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#51] [Announcements]
# Next.js로 기술 블로그 만들기: 블로그는 사드세요... 제발 1편 블로그 구색 갖추기(블로그 포스팅 구현하기) - 정현수 기술 블로그

그냥 웹 사이트가 블로그처럼 보이는 마법

https://junghyeonsu.com/posts/please-buy-the-blog-1/

**Comment by rg5668** [comment]
이미지 부분, 댓글 부분 처리하려고 보고 있었는데, 글 잘 읽었습니다! 참고해서 저도 한 번 만들어봐야겠습니다 ㅎㅎ

**Reply by junghyeonsu** [reply]
오래된 포스팅이긴한데 읽어주셔서 감사합니다..!

**Comment by d5br5** [comment]
좋은 글 잘 읽었습니다 감사합니다.

현수님 글 참고하여 Next.js로 기술블로그를 만들어 보았는데, 
시간 나실때 한 번 방문해주시면 감사하겠습니다..! ㅎㅎ

https://www.d5br5.dev/blog

**Comment by jo4086** [comment]
와..! 얼마전 6개월 node-react 수료를 마쳤고 이제 무엇부터 해야할까 찾아보다가 기본인 블로그부터 만들어보자 라고 생각했는데
꼭 회원가입하게만드는게 아닌 이렇게 블로그만들고 그 자체를 자신만의 완성형 프로젝트로 사용하며 다른 여러가지를 올리 수 있게 만들수가 있었네요!! 이 글을 보고 정신이 번쩍 들었어요..!

**Comment by nopublicsounds** [comment]
감사합ㄴ다... 저도 제 블로그 만드는 데 많이 참고하고 있습니다..
